templates/component-pages/component-name/accessibility.mdx

```diff
@@ -39,3 +48,22 @@ You can have as many Keyboard control blocks as you need.
 
 </KeyboardControl>
 </KeyboardControls>
+
+## WCAG compliance
+
+Statement about WCAG conformance level and testing.
+
+## Known issues
+
+Information or bullet points describing known issues and affected combinations.
+
+## References
+
+- [Link to WCAG success criterion](url)
+- [Link to WCAG technique](url)
+- [Link to ARIA pattern](url)
+- [Link to any other apporprite resources](url)
+
+## Feedback
```

Comment:
Across the site we ask for feedback, we probably need a consistent way to do it

Reply:
I think https://github.com/jpmorganchase/salt-ds/issues/5640 can speak to this, through the backlog.